src/services/mondomarket/index.jsx

```
@@ -264,6 +267,315 @@ export const AddMondoItem = ({ setAddModal, dispatch, setReload, regions }) => {
   );
 };
 
+export const EditMondoItem = ({ setEditModal, dispatch, setReload, item }) => {
```

ESLint

[error] 'setEditModal' is missing in props validation

(react/prop-types)